tests/fips/gnutls/gnutls_base_check.pm

```diff
@@ -18,9 +18,13 @@ use version_utils qw(is_tumbleweed is_sle);
 sub run {
     select_console 'root-console';
 
-    # Install the gnutls / libnettle packages
-    zypper_call('in gnutls libnettle8');
-    zypper_call('info gnutls libnettle8');
+    # Install the gnutls / libnettle packages (pulled as dependency)
+    zypper_call('in gnutls');
```

Comment:
Small suggestion to save 3 lines, but feel free to ignore.
```
my $libnettle = (is_tumbleweed || is_sle('15-SP4+')) ? 'libnettle8' : 'libnettle6';
zypper_call("info gnutls $libnettle");
```

Reply:
Resolved as per discussion below